### DISCUSSION TITLE
Outreachy Applicant

### DISCUSSION THREAD

**Discussion by tolusolaadeyemi** [LesFruitsDefendus/saskatoon-ng#189] [Outreachy]
Hello my name is Tolu from Lagos, Nigeria , I'm an Outreachy  Applicant with experience in python/django, I am very interested in les frutus defendus' fruit harvesting collective. I would be delighted to be contribute to the project.

**Comment by tristanlatr** [comment]
Hello @tolusolaadeyemi,

Thanks for your interest into Saskatoon, 

Look forward for your contribution,

Tristan
